README.md

```diff
@@ -36,17 +36,17 @@ The command for scaffolding a new block. The command will clone the [Element Blo
 
 * * *
 
-### `element publish --name "A NAME TO DISPLAY TO USERS" --category "OPTIONAL CATEGORY" --new-major`
+### `element publish --name "A NAME TO DISPLAY TO USERS" --category "OPTIONAL CATEGORY" --major-version`
```

Comment:
This title is misleading I think. You can not use the major-version option in conjunction with name and category, as major versions use the same name and category of the first published version.

Reply:
Good point.  I'd really like to redo this readme and have the code reside in code blocks and not the section titles. I'll push another commit with just the readme changes.